### ISSUE TITLE
Error with template strings

### ISSUE THREAD

**Issue by IceCreamYou** [Microsoft/TypeScript-Sublime-Plugin#563]
I've been using TypeScript-Sublime-Plugin on a project for a few months now. Today Sublime updated my plugin version and I'm getting a linting error that didn't appear previously and that the TypeScript compiler does not report when I compile my code. The error is:

```
Argument of type 'TemplateStringsArray' is not assignable to parameter of type 'string[]'.
```

This is perplexing for several reasons, not least of which is that `TemplateStringsArray` is defined as `interface TemplateStringsArray extends Array<string>`.

This error appears on code that looks like this:

```ts
safeTag `<div>${variableToEscape}</div>`
```

...where `safeTag` has a type signature like this:

```ts
safeTag(parts: Array<string>, ...subs: Array<string | number | boolean>): string
```

On a related note, the syntax highlighting of template strings seems to have changed at the same time, resulting in interpolated code looking like it's part of the surrounding string instead of being highlighted as code.

**Comment by mottykohn** [comment]
I have the same problem and enclosing in parenthesis gets rid of the error. I believe this is a problem with the Typescript linter not this plugin

**Comment by IceCreamYou** [comment]
I resolved the linting error by changing the template string tag's type signature to:

```ts
safeTag(parts: TemplateStringsArray | Array<string>, ...subs: Array<string | number | boolean>): string;
```

The syntax highlighting in template string interpolations isn't great, but it at least seems to be consistent as of late.

Considering these circumstances and the lack of response to this issue for 1.5 years, I'm going to close this issue.
